tensorflow/c/experimental/filesystem/plugins/gcs/design-proposal.md

```diff
@@ -0,0 +1,101 @@
+# Summary Changes
+| Old Implementation  | New Implementation  |
```

Comment:
Thank you so much for going through the previous implementation and clearing the decisions being made.